drivers/focuser/moonlite.cpp

```diff
@@ -205,6 +205,27 @@ bool MoonLite::readTemperature()
     return true;
 }
 
+bool MoonLite::readTemperatureCoefficient()
+{
+    char res[ML_RES] = {0};
+
+    if (sendCommand(":GC#", res) == false)
+        return false;
+
+    uint8_t coefficient = 0;
+    int rc = sscanf(res, "%hhX", &coefficient);
+    if (rc > 0)
+        // Signed HEX of two digits
+        TemperatureSettingNP[1].setValue(static_cast<int8_t>(coefficient) / 2.0);
```

Comment:
TemperatureSettingNP[Coefficient].setValue(...)

Btw, did you test the rest of the driver since @naheedsa made significant changes today to it.

Reply:
I noticed the code changes that were made after my first addition, so had to adapt a little bit. Itś daytime now, but I tested the driver with mount, telescope and focuser attached, all fields seems to work as before.

Reply:
Tested on different systems today, with different hardware connected, no issues found. When I connect the focuser to a different computer, the temperature coefficient is taken from the focuser with the GC command as the driver initializes, as is the indented behavior.